eval/measure_judged.py

```diff
@@ -67,6 +67,9 @@ def load_run(path: str) -> Dict[str, List[str]]:
                         default=[5, 10, 20, 30],
                         help='Space-separate list of cutoffs. '
                              'E.g.: --cutoffs 5 10 20')
+    parser.add_argument('-q', action='store_true', dest='print_topic',
```

Comment:
we use 120 char line width... so I don't think you need to break up the string?

Reply:
Out of curiosity, do we have linter set up for Python?

Reply:
nope... we decided it (in a separate issue) that it was too heavyweight for a research project...

Reply:
but we do stick to PEP8 as much as possible, we're just not so absolute about it.